src/core/sys/posix/sys/statvfs.d

```diff
@@ -84,6 +84,53 @@ version(CRuntime_Glibc) {
     }
 
 }
+else version(NetBSD)
+{
+    enum  _VFS_MNAMELEN = 1024;
+    enum  _VFS_NAMELEN = 32;
+
+    struct fsid_t
+    {
+       int[2] __fsid_val;
+    }
+
+    struct statvfs_t
+    {
+        c_ulong f_flag;
+        c_ulong f_bsize;
+        c_ulong f_frsize;
+        c_ulong f_iosize;
+        fsblkcnt_t f_blocks;
+        fsblkcnt_t f_bfree;
+        fsblkcnt_t f_bavail;
+        fsblkcnt_t f_bresvd;
+        fsfilcnt_t f_files;
+        fsfilcnt_t f_ffree;
+        fsfilcnt_t f_favail;
+        fsfilcnt_t f_fresvd;
+        ulong f_syncreads;
+        ulong f_syncwrites;
+        ulong f_asyncreads;
+        ulong f_asyncwrites;
+        fsid_t f_fsidx;
+        c_ulong f_fsid;
+        c_ulong f_namemax;
+        int f_owner;
+        int[4] f_spare;
+        char[_VFS_NAMELEN] f_fstypename;
+        char[_VFS_MNAMELEN] f_mntonname;
+        char[_VFS_MNAMELEN] f_mntfromname;
+    }
+
+    enum FFlag
+    {
+        ST_RDONLY = 1,        /* Mount read-only.  */
```

Comment:
Oh, is this comment copied from the header?  If so, remove it.  We don't allow copied comments for potential copyright reasons.